Sources/TensorFlow/Loss.swift

```diff
@@ -98,6 +98,18 @@ public func poissonLoss<Scalar: TensorFlowFloatingPoint>(
     return (predicted - expected * log(predicted)).mean()
 }
 
+/// Returns the Kullback-Leibler divergence between predictions and expectations.
```

Comment:
You should probably say "between expectations and predictions" and also add the actual equation because KL divergence is not symmetric and so this may lead people to thinking the opposite of what's intended.

Reply:
@Shashi456 could you possibly make a new PR addressing this?

Reply:
On it. I'll have it up in a few hours.

Reply:
Great, thanks!